controllers/vspherecluster_controller.go

```diff
@@ -130,7 +94,7 @@ func (r *VSphereClusterReconciler) reconcileNormal(ctx *context.ClusterContext)
 	//   * Downloading OVAs into the content library for any machines that
 	//     use them.
 	//   * Create any load balancers for VMC on AWS, etc.
-	ctx.VSphereCluster.Status.Ready = true
+	ctx.VSphereCluster.Status.Ready = ctx.VSphereCluster.Spec.LoadBalancerConfiguration == nil
 	ctx.Logger.V(6).Info("VSphereCluster is infrastructure-ready")
```

Comment:
Hi @yastij,

Should this log statement be updated based on whether or not the cluster is ready? Before it was always, implicitly ready. Now it depends. I think the log statement should reflect that.